tests/lib/rules/multiline-comment-style.js

```diff
@@ -153,6 +153,30 @@ ruleTester.run("multiline-comment-style", rule, {
         },
         {
             code: `
+                // a comment
+
+                // another comment
+            `,
+            options: ["separate-lines"]
+        },
+        {
+            code: `
+                // a comment
+
+                // another comment
+            `,
+            options: ["bare-block"]
+        },
+        {
+            code: `
+                // a comment
+
+                // another comment
+            `,
+            options: ["starred-block"]
+        },
+        {
+            code: `
```

Comment:
Can you add a test for converting two separate line groups?

```js
/* eslint multiline-comment-style: [error, starred-block] */

// foo
// bar

// baz
// qux
```

should get fixed to:

```js
/*
 * foo
 * bar
 */

/*
 * baz
 * qux
 */
```